templates/base.html

```diff
@@ -57,5 +57,8 @@
 <footer>
   <p>© Cafébabel — 2017</p>
 </footer>
+<div class=preloader>
```

Comment:
I wonder if it wouldn't make sense to use an `id` here instead of `class`. What's your opinion?

Reply:
I never know when it's best to use ID's. I always think _qui peut le plus peut le moins_ ! a class can appear in severals places when an id can't.

Reply:
That's correct. It seems like the loader is covering the entire page and therefore cannot have multiple on a page. That's what made me wonder about this; making it obvious that it can only be unique. Just a thought.

Reply:
ok !